application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -182,4 +178,37 @@ public void onModalSubmitted(ModalInteractionEvent event, List<String> args) {
         throw new UnsupportedOperationException("Not used");
     }
 
+    private boolean isPostAuthor(Member interactionUser, Message message) {
+        if (message.getEmbeds().isEmpty())
+            return false;
+
+        String embedAuthor = Objects
+            .requireNonNull(message.getEmbeds().get(0).getAuthor(),
+                    "embed author for forum post is null")
+            .getName();
+
+        return embedAuthor.equals(interactionUser.getUser().getName());
```

Comment:
if possible, compare user ID instead of name

Reply:
seems like only these properties are available for embed authors, shouldnt be a problem tho username is always unique in discord for everyone.
![image](https://github.com/Together-Java/TJ-Bot/assets/61616007/e8eb7f88-8cb4-4166-860f-5a95a6cba0e0)

Reply:
alright